.github/workflows/main.yml

```diff
@@ -0,0 +1,50 @@
+name: main
+on: [push, pull_request]
+jobs:
+  test:
+    strategy:
+      matrix:
+        node: ["10.x", "12.x", "14.x"]
+        platform: [ubuntu-latest]
+    name: Node v${{matrix.node}} ((${{matrix.platform}}))
+    runs-on: ${{matrix.platform}}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{matrix.node}}
+      - run: yarn install --ignore-engines
```

Comment:
This will speed up a bit the installation, less request to the registry and rely 100% on the lock file.
```
- run: yarn install --ignore-engines --frozen-lockfile
```